weldx/asdf/schemas/weldx.bam.de/weldx/core/transformations/coordinate_system_hierarchy-1.0.0.yaml

```
@@ -42,7 +47,7 @@ properties:
     items:
       $ref: "tag:weldx.bam.de:weldx/core/transformations/coordinate_transformation-1.0.0"
 
-propertyOrder: [name, root_system_name, subsystems, subsystem_data,  coordinate_systems]
+propertyOrder: [name, root_system_name, time_ref, subsystems, subsystem_data,  coordinate_systems]
```

```suggestion
propertyOrder: [name, root_system_name, reference_time, subsystems, subsystem_data,  coordinate_systems]
```
To match the property name?